package-lock.json

```diff
@@ -366,229 +366,6 @@
         }
       }
     },
-    "node_modules/@angular-devkit/build-angular/node_modules/@esbuild/android-arm": {
```

Comment:
Not sure why do you have any changes here but I assume it depends on operating system so I would roll those change back too

Reply:
If we roll back the entire file, we would have the same package-lock.json as before, which causes the release job to fail

Reply:
I doubt this part was responsible as the issue was with `Original error: Pruned lock file creation failed. The following package was not found in the root lock file: @alfresco/js-api@>=8.0.0-alpha.5` and this version was not changed across this file